### PR TITLE
Fix user editing

### LINKED ISSUE #967: Unable to edit new user as an admin
View details in Rollbar: [https://rollbar.com/codefund/codefund/items/453/](https://rollbar.com/codefund/codefund/items/453/)

NoMethodError: private method `select' called for nil:NilClass

```

NoMethodError: private method `select' called for nil:NilClass
  File "/app/vendor/bundle/ruby/2.6.0/gems/tag_columns-0.1.8/lib/tag_columns.rb", line 10, in tag_columns_sanitize_list
  File "/app/vendor/bundle/ruby/2.6.0/gems/tag_columns-0.1.8/lib/tag_columns.rb", line 75, in block (2 levels) in tag_columns
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 429, in instance_exec
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 429, in block in make_lambda
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 201, in block (2 levels) in halting
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 607, in block (2 levels) in default_terminator
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 606, in catch
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 606, in block in default_terminator
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 202, in block in halting
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 514, in block in invoke_before
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 514, in each
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 514, in invoke_before
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 134, in run_callbacks
  File "/app/vendor/bundle/ruby/2.6.0/gems/airbrake-9.5.0/lib/airbrake/rails/active_record.rb", line 20, in run_callbacks
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 827, in _run_validation_callbacks
  File "/app/vendor/bundle/ruby/2.6.0/gems/activemodel-6.0.2.1/lib/active_model/validations/callbacks.rb", line 118, in run_validations!
  File "/app/vendor/bundle/ruby/2.6.0/gems/activemodel-6.0.2.1/lib/active_model/validations.rb", line 337, in valid?
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/validations.rb", line 68, in valid?
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/validations.rb", line 85, in perform_validations
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/validations.rb", line 47, in save
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 315, in block in save
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 375, in block in with_transaction_returning_status
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/database_statements.rb", line 279, in transaction
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 212, in transaction
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 366, in with_transaction_returning_status
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 315, in save
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/suppressor.rb", line 44, in save
  File "/app/vendor/bundle/ruby/2.6.0/gems/scout_apm-2.6.4/lib/scout_apm/instruments/active_record.rb", line 421, in save
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/persistence.rb", line 621, in block in update
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 375, in block in with_transaction_returning_status
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/database_statements.rb", line 281, in block in transaction
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/transaction.rb", line 280, in block in within_new_transaction
  File "/app/vendor/ruby-2.6.5/lib/ruby/2.6.0/monitor.rb", line 235, in mon_synchronize
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/transaction.rb", line 278, in within_new_transaction
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/database_statements.rb", line 281, in transaction
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 212, in transaction
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/transactions.rb", line 366, in with_transaction_returning_status
  File "/app/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/persistence.rb", line 619, in update
  File "/app/app/controllers/users_controller.rb", line 45, in block in update
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/action_controller/metal/mime_responds.rb", line 205, in respond_to
  File "/app/app/controllers/users_controller.rb", line 44, in update
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/action_controller/metal/basic_implicit_render.rb", line 6, in send_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/abstract_controller/base.rb", line 196, in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/scout_apm-2.6.4/lib/scout_apm/instruments/action_controller_rails_3_rails4.rb", line 73, in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/action_controller/metal/rendering.rb", line 30, in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/abstract_controller/callbacks.rb", line 42, in block in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/callbacks.rb", line 135, in run_callbacks
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/abstract_controller/callbacks.rb", line 41, in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/action_controller/metal/rescue.rb", line 22, in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/actionpack-6.0.2.1/lib/action_controller/metal/instrumentation.rb", line 33, in block in process_action
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/notifications.rb", line 180, in block in instrument
  File "/app/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/notifications/instrume

### PR DESCRIPTION
**Pull request by andrewmcodes**
# Bug Fix

## Description

Fix issue with editing/viewing user. Also add ability to create org relationship through OrganizationUser instead of adding `organization_id` to the user object.

Note that the changes to the binstubs came from updating them.

Fixes #967 & #966

### General

- [x] My code follows the style guidelines of this project & the tests are passing
- [x] I have performed a self-review of my own code
- [x] I have updated DB schema annotations or I have not made any DB schema changes

### CodeFund Core Team

- [x] I have added the appropriate milestone if applicable
- [x] I have requested a review from another developer(s)

## Screenshots

image

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def index
 
   def new
     @user = User.new
+    @user.organization_users.build
```
Comment (hopsoft) [inline app/controllers/users_controller.rb]: What is this line doing for us exactly?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -40,6 +41,10 @@ def create
     end
   end
 
+  def edit
+    @user.organization_users.build unless @user.organization_users.exists?
```
Comment (hopsoft) [inline app/controllers/users_controller.rb]: Same question. Are we using the result from `build` anywhere?

File: bin/webpack [hunk 3]
```diff
@@ -7,9 +7,12 @@ require "pathname"
...
 require "bundler/setup"
 
 require "webpacker"
 require "webpacker/webpack_runner"
-Webpacker::WebpackRunner.run(ARGV)
+
+APP_ROOT = File.expand_path("..", __dir__)
+Dir.chdir(APP_ROOT) do
+  Webpacker::WebpackRunner.run(ARGV)
+end
```
Comment (hopsoft) [inline bin/webpack]: Not sure I grok what's happening here yet.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (andrewmcodes) [reply app/controllers/users_controller.rb]: It builds the associated `OrganizationUser` record. Without this, the nested form won't render. See https://github.com/plataformatec/simple_form/wiki/Nested-Models

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (andrewmcodes) [reply app/controllers/users_controller.rb]: See above

File: bin/webpack (hunk 3 above)
Comment (andrewmcodes) [reply bin/webpack]: I just updated the binstubs. `bin/rails webpacker:binstubs`

Comment (andrewmcodes) [reply bin/webpack]: It was an attempt to fix the dev server issues but figured might as well keep the change.

**Pull request merged**